pkg/fleet/installer/setup/defaultscript/default_script.go

```diff
@@ -68,14 +70,18 @@ var (
 )
 
 // SetupDefaultScript sets up the default installation
-func SetupDefaultScript(s *common.Setup) error {
+func SetupDefaultScript(ctx context.Context, s *common.Setup) error {
 	// Telemetry
 	telemetrySupportedEnvVars(s, supportedEnvVars...)
 	if err := exitOnUnsupportedEnvVars(unsupportedEnvVars...); err != nil {
 		return err
 	}
 
 	// Installer management
+	err := packagemanager.RemovePackage(ctx, common.DatadogInstallerPackage)
```

Comment:
add this one in the setup, this was designed as:
- `SetupXXX` configures the setup
- The `common.Setup` object is the one executing everything